.circleci/config.yml

```diff
@@ -34,12 +34,9 @@ jobs:
             - "/usr/local/bin"
             - "/usr/local/lib/python3.7/site-packages"
           key: deps-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
-      - run:
-          name: Run migrations for test
-          command: pipenv run python joplin/manage.py migrate --noinput
       - run:
           name: Run tests
-          command: pipenv run bash .circleci/scripts/run_tests.sh
+          command: pipenv run pytest
```

Comment:
there are a plethoria of other flags we can attach to this command, as well as CLI stuff so if we split up tests later we can run them in different chunks. There is also an option for parallelizing tests